core/market/src/cli.rs

```diff
@@ -0,0 +1,100 @@
+use chrono::{DateTime, Utc};
+use structopt::StructOpt;
+use ya_client::model::market::{agreement::State, Role};
+use ya_core_model::market::{GetAgreement, ListAgreements};
+use ya_service_api::{CliCtx, CommandOutput, ResponseTable};
+use ya_service_bus::{typed as bus, RpcEndpoint};
+
+/// Market management
+#[derive(StructOpt, Debug)]
+pub enum Command {
+    Agreements(AgreementsCommand),
+}
```

Comment:
Like we discussed during demo it would be better to have command `yagna market agreement list` or something like this

Reply:
That's how it already works in this PR (`yagna market agreements list`).